File: Uri/1060.c
#include<stdio.h>
int main()
{
   float numero;
   int valorPositivo=0, contador;
   for (contador=0;contador<=5;contador++)
   {
       scanf("%f",&numero);
       if(numero>=0)
       {
           valorPositivo++;
       }
   } 
   printf("%d valores positivos\n",valorPositivo);
   return 0;
}

File: Uri/1060.cs
using System;

class ValorPositivo
{
    static void Main()
    {
        int valorPositivo=0, contador;
        double v1;
        Console.WriteLine("Digite 6 valores abaixo: ");
        for(contador=1;contador<=6;contador++)
        {
            v1=double.Parse(Console.ReadLine());
            if(v1>=0)
            {
                valorPositivo++;
            }
        }
        Console.Write(valorPositivo + " valores positivos");
    }
}